src/record/record.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Get, Controller, Param, Post, Body } from '@nestjs/common';
+import { RecordService } from './record.service';
+import { Record } from '../db/entity/record';
+import { CreateRecordDto } from './dto/create.record.dto';
+
+/**
+ * Base route is just for various health check endpoints
+ */
+@Controller()
+export class RecordController {
+
+  constructor(private readonly recordService: RecordService) {}
+
+  @Get('records')
```

Comment:
The more standard approach is to have this be the same tense as the rest of the routes, even though it returns multiple records. Then you can also set the 'record' path component in the controller instead of the route to ensure it's the same for all routes used by this Controller.